content/en/community/_index.md

```diff
@@ -17,7 +17,7 @@ main_channels:
       Ask and find answers to your etcd questions.
 community_resources:
   - title: >
-      [<i class="fab fa-google"></i>Google Meet][online]
+      [<i class="fab"></i>Zoom Meeting][online]
```

Comment:
One minor annoyance is that the font awesome 5 brand icons collection we use doesn't include Zoom, so I have just removed the icon.  You can see what I mean in the deploy preview here: https://deploy-preview-720--etcd.netlify.app/community

The list of icons that is available is here https://www.w3schools.com/icons/fontawesome5_icons_brands.asp